src/main/java/controller/LadderController.java

```diff
@@ -0,0 +1,55 @@
+package controller;
+
+import model.LadderGame;
+import model.ladder.Ladder;
+import model.player.Players;
+import model.prize.Prizes;
+import view.InputView;
+import view.ResultView;
+
+public class LadderController {
+
+    private final InputView inputView;
+    private final ResultView resultView;
+
+    private Players players;
+    private Prizes prizes;
+    private Ladder ladder;
+    private LadderGame ladderGame;
+
+    public LadderController(InputView inputView, ResultView resultView) {
+        this.inputView = inputView;
+        this.resultView = resultView;
+    }
+
+    public void runLadderApplication(){
```

Comment:
사소하지만 컨벤션 꾸준히 지켜주세요~!
```suggestion
    public void runLadderApplication() {
```

Reply:
인텔리제이의 자동 정렬 기능도 꼭 활용해보세요!
https://seaotter.tistory.com/111

Reply:
네 주의하겠습니다~!